lib/ai/chat.go

```diff
@@ -18,17 +18,13 @@ package ai
 
 import (
 	"context"
-	"encoding/json"
-	"errors"
-	"io"
-	"strings"
 
 	"github.com/gravitational/trace"
 	"github.com/sashabaranov/go-openai"
 	"github.com/tiktoken-go/tokenizer"
-)
 
-const maxResponseTokens = 2000
```

Comment:
This was used to avoid sometimes excessively large responses. With the new agent model and prompt setup, output size is better controlled and this only prevents the occasional large useful response.